kcp/types.go

```diff
@@ -0,0 +1,24 @@
+package kcp
+
+import (
+	"k8s.io/apimachinery/pkg/types"
+)
+
+// NamespaceReference holds a reference to a specific namespace within a cluster using KCP
```

Comment:
you added periods to the other comments like this why not here?

Reply:
I'll fix it.